zzzz_modular_occulus/code/modules/client/preference_setup/loadout/lists/suits.dm

```diff
@@ -0,0 +1,11 @@
+// Suit slot
+/datum/gear/suit
```

Comment:
This bit doesn't need to be copypasta'd! You can comment it out if you wanna keep it for future reference without looking for the original file.

Reply:
...I got entirely confused on how the load out system works, and I copy pasted it without thinking. Whoops.